src/plugins/home/server/plugin.ts

```diff
@@ -36,13 +39,17 @@ export class HomeServerPlugin implements Plugin<HomeServerPluginSetup, HomeServe
     const router = core.http.createRouter();
     registerRoutes(router);
 
+    this.customIntegrations = plugins.customIntegrations;
     return {
-      tutorials: { ...this.tutorialsRegistry.setup(core) },
+      tutorials: { ...this.tutorialsRegistry.setup(core, plugins.customIntegrations) },
       sampleData: { ...this.sampleDataRegistry.setup(core, plugins.usageCollection) },
     };
   }
 
   public start(): HomeServerPluginStart {
+    if (!this.customIntegrations) {
+      throw new Error('Cannot start home server. CustomIntegrations plugin missing.');
+    }
```

Comment:
NIT: plugin is defined as `required`, the only thing you're asserting here is that `setup` was called before `start`

Reply:
My apologies. this was cruft copied over from an earlier POC (https://github.com/elastic/kibana/pull/112330), which required access to this plugin in start. Removing altogether.